pkg/reconciler/apis/permissionclaimlabel/permissionclaimlabel_reconcile.go

```diff
@@ -159,18 +161,33 @@ func (c *controller) reconcilePermissionClaims(ctx context.Context, apiBinding *
 		}
 	}
 
-	klog.V(2).Infof("Adding permission claims %v for APIBinding %s", addedPermissionClaims, fmt.Sprintf("%s|%s", lc, apiBinding.Name))
+	if len(addedPermissionClaims) != 0 {
+		klog.V(2).Infof("Adding permission claims %v for APIBinding %s", addedPermissionClaims, fmt.Sprintf("%s|%s", lc, apiBinding.Name))
```

Comment:
Never use `fmt.Sprintf` in a logging argument. Put the pattern into the first arg. Otherwise, you allocate without need.

Reply:
removed